package.json

```diff
@@ -12,13 +12,22 @@
     "eslint:strict": "STRICT=1 eslint -c src/.eslintrc.js 'src/**/*.ts?(x)'",
     "prettier": "prettier --write",
     "preview": "vite preview",
-    "start": "vite"
+    "start": "vite",
+    "test:exact": "jest --",
+    "test:watch": "jest --watch  --verbose",
+    "test": "jest --watchAll=false"
   },
   "dependencies": {
     "react": "^18.2.0",
-    "react-dom": "^18.2.0"
+    "react-dom": "^18.2.0",
```

Comment:
Any reason we shouldn't pin these both to `18.2.0`?

Reply:
no reason, I think the Vite project was just generated like that with the `^` dependencies! I pinned them 😈